sample/src/main/java/com/github/terrakok/androidcomposeapp/screens/SampleScreen.kt

```diff
@@ -45,8 +45,7 @@ import kotlinx.parcelize.Parcelize
 @Parcelize
 class SampleScreen(
     private val i: Int,
-    override val screenKey: ScreenKey = generateScreenKey()
-) : Screen {
+) : Screen(screenKey = generateScreenKey()) {
```

Comment:
unfortunately in this case it not gonna be parcelized, so we must put in the constructor